download.sh

```diff
@@ -41,29 +41,25 @@ fi
 
 
 # configure nvidia apt repo to cache packages
-curl -fsSLO https://nvidia.github.io/nvidia-docker/gpgkey
+curl -fsSLO https://nvidia.github.io/libnvidia-container/gpgkey
 gpg --dearmor -o aptnvidia.gpg gpgkey
 mv aptnvidia.gpg /etc/apt/trusted.gpg.d/aptnvidia.gpg
-curl -fsSL https://nvidia.github.io/nvidia-docker/ubuntu${VERSION_ID}/nvidia-docker.list -o /etc/apt/sources.list.d/nvidia-docker.list
+curl -fsSL https://nvidia.github.io/libnvidia-container/stable/deb/nvidia-container-toolkit.list -o /etc/apt/sources.list.d/nvidia-container-toolkit.list
 
 apt update
+chmod 644 /etc/apt/trusted.gpg.d/*
+
 
-# download nvidia debian packages for nvidia-container-runtime compat
 for apt_package in $NVIDIA_PACKAGES; do
     apt-get download ${apt_package}=${NVIDIA_CONTAINER_TOOLKIT_VER}*
     mv ${apt_package}_${NVIDIA_CONTAINER_TOOLKIT_VER}* /opt/gpu
 done
-apt-get download nvidia-container-runtime=${NVIDIA_CONTAINER_RUNTIME_VERSION}*
 
-# move debs to permanent cache
-mv nvidia-container-runtime_${NVIDIA_CONTAINER_RUNTIME_VERSION}* /opt/gpu
-
-# nvidia-docker2 for docker runtime
-apt-get download nvidia-docker2=${NVIDIA_DOCKER_VERSION}
-mkdir -p /tmp/nvidia-docker2
-dpkg-deb -R ./nvidia-docker2_${NVIDIA_DOCKER_VERSION}_all.deb /tmp/nvidia-docker2
-mkdir -p /opt/gpu/nvidia-docker2_${NVIDIA_DOCKER_VERSION}
-cp -r /tmp/nvidia-docker2/usr/* /opt/gpu/nvidia-docker2_${NVIDIA_DOCKER_VERSION}/
+# download nvidia debian packages for nvidia-container-runtime compat
```

Comment:
Is there a reason that we're downloading the packages twice?

Reply:
nice find ! @elezar , @ganeshkumarashok  copy paste error ?